reposilite-backend/src/main/kotlin/com/reposilite/console/application/ConsolePlugin.kt

```diff
@@ -35,13 +36,28 @@ import com.reposilite.plugin.reposilite
 import com.reposilite.web.api.HttpServerInitializationEvent
 import com.reposilite.web.api.RoutingSetupEvent
 import com.reposilite.web.application.WebSettings
+import java.util.concurrent.TimeUnit
 
 @Plugin(name = "console", dependencies = [ "shared-configuration", "failure", "access-token", "authentication" ])
 internal class ConsolePlugin : ReposilitePlugin() {
 
     override fun initialize(): Facade {
         val sharedConfigurationFacade = facade<SharedConfigurationFacade>()
         val consoleFacade = ConsoleComponents(this, facade()).consoleFacade()
+        val client = ConsoleSseHandler(
+            journalist = reposilite().journalist,
+            accessTokenFacade = facade(),
+            authenticationFacade = facade(),
+            forwardedIp = sharedConfigurationFacade.getDomainSettings<WebSettings>().computed { it.forwardedIp }
+        )
+
+        val watcher = reposilite().scheduler.scheduleWithFixedDelay({
+            // use an iterator instead of forEach to avoid CME
+            val iterator = client.users.iterator()
+            while (iterator.hasNext()) {
+                iterator.next().key.sendComment("ping")
+            }
+        }, 0, 1, TimeUnit.SECONDS)
```

Comment:
I don't think that such frequent ping is mandatory here, so maybe we can increase the interval a little bit? For WS we used 5s, so that could be an option

Reply:
Yeah, looks like a workaround to some underlying issue. It's logged without formatting, so I guess the error happens insde the logging impl - I've found regular HashMap being used here:

* https://github.com/reposilite-playground/journalist/blob/main/journalist/src/main/java/com/reposilite/journalist/backend/PublisherLogger.java#L14

So it might be a good idea to fix it here and see if it helps. In the meantime, let's keep a "proper" approach here (5s) and ignore the error.

Reply:
I've just pushed a newer version of Journalist, you can try to bump it to 1.0.12 to check if it helped.

Reply:
That fixed the CME errors :)

Reply:
Nice! :)